velox/exec/Task.cpp

```diff
@@ -1013,15 +1013,15 @@ void Task::updateBroadcastOutputBuffers(int numBuffers, bool noMoreBuffers) {
     std::lock_guard<std::mutex> l(mutex_);
     if (noMoreBroadcastBuffers_) {
       // Ignore messages received after no-more-buffers message.
-      return;
+      return true;
```

Comment:
Should we return false here since we are not updating the buffers?

Reply:
When `noMoreBroadcastBuffers_ == true`, the current logic is ignoring or it is just a no-op, so I feel returning true helps the caller. It is easier for caller to make out if the error case is coming from task being deleted and can take necessary action.